pilot/pkg/autoregistration/controller_test.go

```diff
@@ -412,6 +409,19 @@ func checkEntryOrFail(
 	}
 }
 
+func checkEntryOrFailAfter(
+	t test.Failer,
+	store model.ConfigStoreController,
+	wg config.Config,
+	proxy *model.Proxy,
+	node *core.Node,
+	connectedTo string,
+	after time.Duration,
+) {
+	time.Sleep(after)
+	checkEntryOrFail(t, store, wg, proxy, node, connectedTo)
```

Comment:
It always sleep that long. Why not polling periordically? I think we have some utility 
 function like wait.PollUntil

Reply:
This is case we have to sleep for short duration to simulate fast reconnect and check entry. It is not to poll and check